src/@polymer/lit-element-decorators.ts

```diff
@@ -0,0 +1,25 @@
+export type LitElementPropertyType = BooleanConstructor|DateConstructor|NumberConstructor|StringConstructor|ArrayConstructor|ObjectConstructor;
+
+export interface PropertyOptions {
+  type: LitElementPropertyType;
+}
+
+export const customElement = (tagname: keyof HTMLElementTagNameMap) => {
+  return <T extends { new (...args: any[]): HTMLElement }>(clazz: T) => {
+    window.customElements.define(tagname, clazz);
+    return clazz;
+  };
+}
+
+export const property = (options: PropertyOptions | LitElementPropertyType) => {
```

Comment:
Could we add optional support for reflect-metadata? Perhaps something similar to https://github.com/pshihn/alit-element/blob/3d1cf69434295ed1037e2d8a30833960952a0703/src/alit-element-decorators.ts#L31

That would be great for code sanity as there will be no need maintain consistency between decorator and type definition